tests/javascript/calendar/spec.js

```diff
@@ -21,19 +21,21 @@ define(['jquery', 'testsRoot/calendar/spec-setup', 'jasmineJquery'], function ($
 			JoomlaCalendar.init(element);
 		});
 
-		it('Should have calendar element under the input element', function () {
+		it('Should have calendar element under the input element', function (done) {
 			expect($('body')).toContainElement('.js-calendar');
```

Comment:
Even for Unit Tests, it would be better if you can use Vanilla JavaScript at least in places, which won't increase the complexity.

Reply:
Yeah sure will do